src/wallet/walletdb.h

```diff
@@ -73,7 +73,11 @@ class CHDChain
     static const int VERSION_BASIC = 1;
     static const int VERSION_WITH_BIP44 = 10;
     static const int CURRENT_VERSION = VERSION_WITH_BIP44;
+#ifdef ENABLE_EXODUS
+    static const int N_CHANGES = 4; // standard = 0/1, mint = 2, exodus = 3
```

Comment:
We should make it always 4 without conditional; otherwise the data in wallet file will not consistent.